extension/framework/goplugin/runner/runner.go

```diff
@@ -180,9 +182,23 @@ func (testRunner *TestRunner) runSingle(t ginkgo.GinkgoTestingT, reporter *Repor
 	// create env
 	beforeStartHook := func(env *goplugin.Environment) error {
 		// db
-		dbFileName := dbBaseFileName + "_" + uuid.NewV4().String()
-		dbConnString := fmt.Sprintf("file:%s?mode=memory&cache=shared", dbFileName)
-		db, err := gohan_db.ConnectDB("sqlite3", dbConnString, gohan_db.DefaultMaxOpenConn, options.Default())
+		dbType := ""
+		dbConn := ""
+
+		if os.Getenv("MYSQL_TEST") == "true" {
+			dbType = "mysql"
+			if dsn, ok := os.LookupEnv("MYSQL_DSN"); ok {
+				dbConn = dsn
+			} else {
+				dbConn = "root@/gohan_test"
```

Comment:
Is hardcode ok?

Reply:
It's not the best solution here. There were a dozen of similar places in code where connection was hardcoded. I added a commit which contains a common file for local database connection and clears Connect and InitSchema functions naming.